src/util/general.js

```diff
@@ -0,0 +1,144 @@
+'use strict'
```

Comment:
Cant we name this file to something that is more applicable.

I.e. `src/util/objects.js` <-- assuming all its util functions are applicable to objects. Let me know what you think?

Reply:
I dont think naming it object is wish cause when referencing in code it could be confused with the Object type in javascript, and will make it a little less readable